setup.cfg

```diff
@@ -78,6 +81,12 @@ omit =
     *bin/pyproject-build
     *bin\pyproject-build.exe
 
+[coverage:report]
+exclude_lines =
+    \#\s*pragma: no cover
+    ^\s*raise NotImplementedError\b
+    ^if __name__ == ['"]__main__['"]:$
```

Comment:
Why? We test this.

Reply:
Yes. Inside the ``__main__`` file. This makes that line covered.

Reply:
But we test that via integration tests, it should already be covered :thinking: